### ISSUE TITLE
Selecting an entry with `--parameters` breaks commandline execution

### ISSUE THREAD

**Issue by doque** [atuinsh/atuin#1086]
I'm selecting a command that passes parameters using `--`:

```
--data-raw '{"lights":[{"on":1}],"numberOfLights":1}'
```

![CleanShot 2023-07-05 at 11 47 30](https://github.com/ellie/atuin/assets/1480751/3768725c-539f-470c-b383-b41d1501b310)
e
This seems to break the passing to `commandline`:

```
~ commandline: --data-raw '{"lights":[{"on":1}],"numberOfLights":1}': unknown option

- (line 21):
        commandline -r $h
        ^
in function '_atuin_search' with arguments '--shell-up-key-binding'
	called on line 36 of file -
in function '_atuin_bind_up'

(Type 'help commandline' for related documentation)
```

**Comment by conradludgate** [comment]
Thanks for raising. What version of atuin are you using and what shell?

**Comment by doque** [comment]
I got these versions running:

```
➜  ~ fish --version
fish, version 3.5.1
➜  ~ atuin --version
atuin 15.0.0
```

**Comment by thigg** [comment]
I get the same error with fish and atuin with a command that was originally multiline in the shell

```
docker run -idt
--network host
--privileged
--name abc
```

selecting this in atuin leads to
```

- (line 21):
        commandline -r $h
        ^
in function '_atuin_search' with arguments '--shell-up-key-binding'
	called on line 36 of file -
in function '_atuin_bind_up'
```

**Comment by ellie** [comment]
This should have been resolved by #1237
